### PR TITLE
docs: clear sourceLabels

### LINKED ISSUE #1930: sourceLabels doc. seems unclear in docs/reference/config/networking/virtual-service/index.html
In the section currently at https://istio.io/latest/docs/reference/config/networking/virtual-service/#L4MatchAttributes-source_labels,
where the text says:

> One or more labels that constrain the applicability of a rule to workloads with the given labels.

it's not clear whether that "workloads with the given labels" applies to the workloads for _calling_ services or to workloads for _called_ services.

(Yes, the name `sourceLabels` _suggests_ that it's for calling services, but the text doesn't seem to actually confirm that.)

(Also, the wording "labels that constrain the applicability of a rule to workloads" is quite abstract and doesn't really specify much.  It should probably mention something like "selecting workloads with the corresponding labels" (maybe also referring to a similar member, such as "&lt;something>'s `selector`" (or `workloadSelector`)).)

### PR DESCRIPTION
**Pull request by daixiang0**
Fix #1930

### CODE REVIEW

File: networking/v1alpha3/virtual_service.proto [hunk 1]
```diff
@@ -1070,7 +1070,7 @@ message HTTPMatchRequest {
   uint32 port = 6;
 
   // One or more labels that constrain the applicability of a rule to
-  // workloads with the given labels. If the VirtualService has a list of
+  // workloads with the given labels for calling services. If the VirtualService has a list of
```
Comment (linsun) [inline networking/v1alpha3/virtual_service.proto:1073]: How about for source services?

Comment (daixiang0) [reply networking/v1alpha3/virtual_service.proto:1073]: Update.

**Comment by daixiang0** [comment]
@linsun could you help add release-note-none label?

**Comment by daixiang0** [comment]
@linsun could it be merged?

**Comment by daixiang0** [comment]
@linsun @dcberg @nrjoddar could it be merged?

File: networking/v1alpha3/virtual_service.proto [hunk 2]
```diff
@@ -1070,7 +1070,7 @@ message HTTPMatchRequest {
   uint32 port = 6;
 
   // One or more labels that constrain the applicability of a rule to
-  // workloads with the given labels. If the VirtualService has a list of
+  // workloads with the given labels for source services. If the VirtualService has a list of
```
Comment (nrjpoddar) [inline networking/v1alpha3/virtual_service.proto:1073]:
```suggestion
  // source (client) workloads with the given labels for source services. If the VirtualService has a list of
```

Comment (daixiang0) [reply networking/v1alpha3/virtual_service.proto]: Done. @nrjpoddar

File: networking/v1alpha3/virtual_service.proto [hunk 3]
```diff
@@ -1069,8 +1069,8 @@ message HTTPMatchRequest {
   // in these cases it is not required to explicitly select the port.
   uint32 port = 6;
 
-  // One or more labels that constrain the applicability of a rule to
-  // workloads with the given labels. If the VirtualService has a list of
+  // One or more labels that constrain the applicability of a rule to source (client) workloads
+  // with the given labels for source services. If the VirtualService has a list of
```
Comment (nrjpoddar) [inline networking/v1alpha3/virtual_service.proto:1073]:
```suggestion
  // with the given labels. If the VirtualService has a list of
```

**Review by nrjpoddar — COMMENTED** [review]
Sorry I missed removing "for source services" in my first feedback.

File: networking/v1alpha3/virtual_service.proto (hunk 3 above)
Comment (daixiang0) [reply networking/v1alpha3/virtual_service.proto]: Done.

**Comment by daixiang0** [comment]
Thanks for quick input.

**Review by nrjpoddar — APPROVED** [review]

**Comment by daixiang0** [comment]
@nrjpoddar please review

**Comment by nrjpoddar** [comment]
It's approved from my side; you need a second approver to get it merged. Thanks for the PR!

**Comment by daixiang0** [comment]
Thanks for review! @nrjpoddar 

Nicely ask review @dcberg  @duderino  @louiscryan @smawson

**Review by howardjohn — APPROVED** [review]

**Pull request merged**